fedbiomed/common/optimizers/declearn.py

```diff
@@ -14,12 +14,14 @@
 )
 
 from declearn.optimizer.modules import (
+    AuxVar,
     OptiModule,
     AdaGradModule,
```

Comment:
I added them here so that in tests we can import them from `fedbiomed` rather than from `declearn`. I have not strong opinion about it, but it felt coherent with the fact that other declearn-defined classes are imported from fedbiomed (that explicitly filters what is available).